webapp/src/ts/services/enketo.service.ts

```diff
@@ -309,6 +290,10 @@ export class EnketoService {
           return Promise.reject(new Error(JSON.stringify(loadErrors)));
         }
       })
+      .then(() => this.languageService.get())
```

Comment:
We already have the language available in the previous block, via `options.language`. There's no need to request it again.